Makefile.am

```diff
@@ -138,6 +138,16 @@ lib_libcommon_a_SOURCES = \
     lib/tpm2_tcti_ldr.c \
```

Comment:
Is there any need  to have #769 twice in the commit message?

Reply:
Nope.

Reply:
Actually it's github messing it up, the verbatim text is:
```
The tools will be consuming this for the 4.0 release under
ticket #769:
  - https://github.com/tpm2-software/tpm2-tools/issues/769
```